.travis.yml

```diff
@@ -1,45 +1,71 @@
-language: python
+language: c
 
 env:
-  - PYTHON_VERSION=3.6
+  - PYTHON_VERSION=3.7
 
-addons:
-  apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    packages:
-    - libopenmpi-dev
+matrix:
+  include:
+      # Linux
+      - os: linux
+        name: "Linux full"
+        addons:
+          apt:
+            sources:
+              - ubuntu-toolchain-r-test
+            packages:
+              - libopenmpi-dev
+              - openmpi-bin
+      # OSX
+      - os: osx
+        name: "MacOS full"
 
 
 before_install:
-  - wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+        wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+        wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
+    fi;
   - chmod +x miniconda.sh
-  - ./miniconda.sh -b -p /home/travis/miniconda
-  - export PATH=/home/travis/miniconda/bin:$PATH
+  - ./miniconda.sh -b -p ${HOME}/miniconda
+  - export PATH=${HOME}/miniconda/bin:$PATH
   - conda update --yes --quiet conda
 
 install:
     - install_prefix=~
     - conda create -n testenv --yes pip python=${PYTHON_VERSION}
     - source activate testenv
     - conda install --yes scipy numpy matplotlib
+    - |
+      if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        conda install --yes mpi4py openmpi
```

Comment:
how do I try your PR on mac? I tried this command to install and it's stuck for the last 15 minutes ...

Reply:
For the most part these commands work for me. Very well could have a different build environment though. Could you send me any output you get after this?

Note, I had to do:
```
conda activate testenv
# fix on osx for coverals
pip install urllib3==1.24
pip install coverage coveralls
```